src/app/Http/Controllers/CountryController.php

```diff
@@ -0,0 +1,36 @@
+<?php
+
+
+namespace App\Http\Controllers;
+
+
+use App\Services\CoronaStatusService;
+use Illuminate\Http\Request;
+use Illuminate\View\View;
+
+class CountryController
+{
+    private CoronaStatusService $coronaStatusService;
+
+    public function __construct(CoronaStatusService $coronaStatusService)
+    {
+        $this->coronaStatusService = $coronaStatusService;
+    }
+
+    public function show(Request $request): View
+    {
+        $request->validate([
+            'country' => ['required', 'min:2','max:2', 'string'],
+        ]);
+
+        $country = mb_strtolower($request->get('country'));
+
+        $responseText = $this->coronaStatusService->isCountryAffected($country) ? 'Affected' : 'Not affected';
```

Comment:
```suggestion
        $isAffected = $this->coronaStatusService->isCountryAffected($country);
```

Negerai čia padarėm, kad įdėjom logiką bei tekstą į kontrolerį. Pakanka, kad iš serviso gauni atsakymą `true|false`, toliau perduok jį BLADE ir aprašyk `lang` failą (paaiškinimą rasi `country.index` blade faile).